DependencyInjection/Factory/Loader/FileSystemLoaderFactory.php

```diff
@@ -61,4 +63,18 @@ public function addConfiguration(ArrayNodeDefinition $builder)
                 ->end()
             ->end();
     }
+
+    /**
+     * @param string $reference
+     *
+     * @return Reference
+     */
+    private function createLocatorReference($reference)
+    {
+        return new Reference(
+            sprintf('liip_imagine.binary.locator.%s', $reference),
+            ContainerInterface::EXCEPTION_ON_INVALID_REFERENCE,
+            SymfonyFramework::hasDefinitionSharedToggle()
```

Comment:
The third argument (strict) was removed in symfony 3.0.
Should we use an switch like

```php
if (symfony 2.x) {
    return new Reference(
        sprintf('liip_imagine.binary.locator.%s', $reference),
            ContainerInterface::EXCEPTION_ON_INVALID_REFERENCE,
            false
    );

return new Reference(sprintf('liip_imagine.binary.locator.%s', $config['locator']);
}
```